locales/en/email-strings.ftl

```diff
@@ -129,3 +129,26 @@ more-products-vpn = Protection for your whole device, on every device.
 more-products-cta-vpn = Get {-product-name-vpn}
 more-products-relay = Hide your real email address to help protect your identity.
 more-products-cta-relay = Get {-product-name-relay}
+
+## 2022 email template. HTML tags should not be translated, e.g. `<a>`
+
+email-2022-unsubscribe = You’re receiving this automated email as a subscriber of { -product-name }. <br>Feel free to change your email preferences at any time <a href="{ $unsubscribe-url }">here</a>.
+# Have I Been Pwned attribution
+email-2022-hibp-attribution = Breach data provided by <a href="{ $hibp-url }" rel="noopener">{-brand-HIBP}</a>
```

Comment:
```suggestion
email-2022-hibp-attribution = Breach data provided by <a href="{ $hibp-url }" rel="noopener">{ -brand-HIBP }</a>
```

Missed this one. It's not broken, but it would be nice to write the syntax consistently (leading and trailing space within {}).